tests/test_export_custom_exporter.py

```diff
@@ -18,13 +18,13 @@ def export(self, data_contract, model, server, sql_server_type, export_args) ->
         return str(result)
 
 
-exporter_factory.register_exporter("custom", CustomExporter)
+exporter_factory.register_exporter("custom_exporter", CustomExporter)
```

Comment:
`custom` export_format is used in this feature, so I changed `CustomExporter` tests.